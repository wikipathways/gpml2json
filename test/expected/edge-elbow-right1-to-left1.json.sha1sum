d923491216abdbf1b334a3e4f196b7b635d4c5b1 ?-
