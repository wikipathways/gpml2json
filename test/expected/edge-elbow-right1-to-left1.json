{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e4b31","a739c","d3ec7"],"drawAs":"rect","gpmlElementName":"Pathway","height":783.1967827425314,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1045.87076968672,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/b64fc7edf41f58872798ecd2cf8774b5a91cad3e578433c67515dcb7dced056f"},"entityMap":{"a739c":{"id":"a739c","textContent":"GeneProduct","wpType":"GeneProduct","x":180.3444354561113,"y":115.56929497810864,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33011,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d3ec7":{"id":"d3ec7","textContent":"0023","wpType":"GeneProduct","x":15.28604108781412,"y":60.00623839468847,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33012,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e4b31":{"id":"e4b31","drawAs":"ElbowLine","borderWidth":1,"zIndex":33010,"points":[{"isAttachedTo":"d3ec7","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":95.28604108781411,"y":70.00623839468847,"orientation":[1,0]},{"y":97.78776668639856,"x":115.28604108781411},{"isAttachedTo":"a739c","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":180.34443545611134,"y":125.56929497810864,"orientation":[1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["d3ec7","a739c"]}}}
